Solutions/Azure Active Directory/Analytic Rules/AccountCreatedDeletedByNonApprovedUser.yaml

```diff
@@ -26,18 +26,18 @@ query: |
   AuditLogs
   | where OperationName == "Add user" or OperationName == "Delete user"
   | where Result == "success"
-  | extend InitiatingUser = tostring(parse_json(tostring(InitiatedBy.user)).userPrincipalName)
+  | extend InitiatingUser = tostring(InitiatedBy.user.userPrincipalName)
   | where InitiatingUser has_any (nonapproved_users)
```

Comment:
Again whilst we are doing this work do we want to replace this static list of nonapproved users with a solution Watchlist instead - makes it more scalable and usable for customers

Reply:
We don't but we could add an empty one for users to populate, maybe rather than have non-approved users we could have a Watchlist for "Approved User Account Management Accounts" and change the query to look for users not in this list.

Reply:
@petebryan I will add this to our backlog to improve all the queries using this watchlist.